client/blocks/site-address-changer/dialog.jsx

```diff
@@ -68,11 +81,46 @@ class SiteAddressChangerConfirmationDialog extends PureComponent {
 				action: 'confirm',
 				label: translate( 'Change site address' ),
 				onClick: this.onConfirm,
-				disabled: ! this.state.isConfirmationChecked,
+				disabled: ! this.state.isConfirmationChecked || ! this.state.currentAddressOption,
 				isPrimary: true,
 			},
 		];
 
+		const options = [
+			{
+				id: 'redirect',
+				answerText: translate(
+					'Redirect to {{strong}}%(newSiteAddress)s{{/strong}} (%(price)s per year)',
+					{
+						components: {
+							strong: <strong />,
+						},
+						args: {
+							newSiteAddress: newDomainName + newDomainSuffix,
+							price: siteRedirectPrice,
+						},
+					}
+				),
```

Comment:
When loading the dialog, the prop type checker yields an error:

![change-site-address-dev-console-error](https://user-images.githubusercontent.com/18705930/132250965-494b2e5d-2c29-4a23-b2a3-5ba98d02e12a.png)

Here's how the component props are in that moment:

![change-site-address-dev-console-react-component-error](https://user-images.githubusercontent.com/18705930/132251033-e474041f-2760-4f45-b45e-4c42dbec6a5c.png)

This happens because `translate` returns an array in this case (a React Fragment). 

While this is not the optimal solution, it can be easily solved by removing the `components` prop, thus returning a string. 
Still, I think the best way to fix this would be to change the prop type on the `MultipleChoiceQuestion` component.
Leaving it as-is will only show the error on the console. The component is rendered just fine 😸